packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -34,6 +34,7 @@ Requires: python3-requests
 
 Requires: %{?scl_prefix}rubygem(smart_proxy_dynflow_core) >= 0.1.5
 Requires: %{?scl_prefix}rubygem(foreman_ansible_core)
+Requires: ansible-collection-theforeman-foreman
```

Comment:
Does this mean we can drop the dependency on python(3)-requests and let the collection pull it in?

Reply:
indeed, it does.

Reply:
Can you drop lines 29-33 then?